docs/install-upgrade/install.md

```diff
@@ -84,7 +101,15 @@ This control node should be given a static IP address. Either a lease or statica
 
 ### Configure Management Network
 
-The control node is dual-homed. It has a 10GbE interface that connects to the management network. The other link called `external` in the `fab.yaml` file is for the customer to access the control node. The management network is for the command and control of the switches that comprise the fabric. This management network can be a simple broadcast domain with layer 2 connectivity. The control node will run a DHCP and small http servers. The management network is not accessible to machines or devices not associated with the fabric.
+The control node is dual-homed; the two networks it connects to are called respectively `management`
+and `external` in the `fab.yaml` file.
+The `management` network is for the command and control of the switches that comprise the fabric. It
+can be a simple broadcast domain with layer 2 connectivity. The management network is not accessible
+to machines or devices not associated with the fabric; it is a private, exclusive network. The control
+node has a 10GbE interface that connects to the management network, and it runs a DHCP and a small http
+server.
+The `external` network on the other hand is for the customer to access the control node via their local
+IT network. It is meant to give ssh access to the control node host operating system.
```

Comment:
- What's “command and control” (does it bring more precision than from simply “control”)?

- (I know you didn't add it but) What's the HTTP server for, and how does the related sentence connect to the previous one?

- Who is “the customer”? This is an open-source project, you probably mean “the user” :slightly_smiling_face:

```suggestion
The `management` network is for controlling the switches that comprise the fabric. It
can be a simple broadcast domain with layer 2 connectivity. The management network is
not accessible to machines or devices not associated with the fabric; it is a private,
exclusive network. The control node connects to the management network via a 10 GbE
interface. It runs a DHCP server, as well as a small HTTP server.

The `external` network allows the user to access the control node via their local
IT network. It provides SSH access to the host operating system on the control node.
```